docs-data/rpk-overrides.schema.json

```
@@ -0,0 +1,724 @@
+                    "not": { "required": ["selfHostedOnly"] }
+                  },
+                  {
+                    "required": ["selfHostedOnly"],
+                    "not": { "required": ["cloudOnly"] }
+                  }
+                ]
+              }
+            ]
+          }
+        },
+        "pageAliases": {
```

Biome (2.4.16)

[error] The key pageAliases was already declared.

(lint/suspicious/noDuplicateObjectKeys)